pytest.ini

```
@@ -8,5 +8,7 @@ filterwarnings =
     ignore:Using or importing the ABCs from:DeprecationWarning:pyreadline
     # onnx calling deprecated API of google.protobuf
     ignore:Call to deprecated create function:DeprecationWarning:onnx
+    # numpy 1.20 deprecation for np.object referencing
+    ignore:.*is a deprecated alias for the builtin:DeprecationWarning
```

Let's make sure that deprecations are not coming from our code 😃 
```suggestion
    ignore:.*is a deprecated alias for the builtin:DeprecationWarning:onnx
```